weed/command/filer_sync.go

```diff
@@ -427,6 +427,11 @@ func genProcessFunction(sourcePath string, targetPath string, excludePaths []str
 			return nil
 		}
 
+		// handle truncate entries
+		if filer_pb.IsUpdate(resp) && resp.EventNotification.DeleteChunks && dataSink.IsIncremental() {
```

Comment:
`resp.EventNotification.DeleteChunks` is only used in sink to another filer.

Not sure what does this mean. It sounds like "for incremental mode, do not handle updates".

Reply:
> Not sure what does this mean. It sounds like "for incremental mode, do not handle updates".

May need more time to reproduce. 
I found the following calls:
<img width="923" alt="image" src="https://github.com/seaweedfs/seaweedfs/assets/9497591/437373f6-d81a-42b7-b6b5-e6e7d51b4c00">

Do not fit where signatures is nill and newEntry or oldEntry is nill and deleteChunks is false.
And that remains:
1. https://github.com/seaweedfs/seaweedfs/blob/17710e1ecbd36409fa5b14861c025aacf943b6fd/weed/filer/filer.go#L245
2. https://github.com/seaweedfs/seaweedfs/blob/17710e1ecbd36409fa5b14861c025aacf943b6fd/weed/server/filer_grpc_server.go#L195